### ISSUE TITLE
Is it necessary to generate geojson for each province(or same level district) in China?

### ISSUE THREAD

**Issue by LaoshuBaby** [ideditor/location-conflation#34]
What preparatory work is needed to complete this work?
Where should we post? [NSI](github.com/osmlab/name-suggestion-index)? [LC](https://github.com/ideditor/location-conflation)? [CC](https://github.com/ideditor/country-coder)?

**Comment by bhousel** [comment]
We have some issues here:
https://github.com/ideditor/country-coder/issues/19
https://github.com/ideditor/country-coder/issues/26
https://github.com/osmlab/name-suggestion-index/issues/4784

Basically, we don't want to add too much to country-coder because it gets bundled into iD and we want to conserve space there.

I've been ok with adding these features to name-suggestion-index because those features get fetched and loaded just after iD starts.  I looked for state features and [I tried](https://github.com/osmlab/name-suggestion-index/issues/4784#issuecomment-743241895) merging them and making the seamless, but I couldn't quickly figure out how to do it. 

I could imagine adding features for all the states and provinces where people are doing a lot of mapping.
